tests/test_utils.py

```diff
@@ -179,7 +179,7 @@ def test_guess_mimetype(name, mime):
 
 
 def test_guess_mimetype_patch():
-    """ Test WIN32_PATCHED_MIME_TYPES. """
+    """Test WIN32_PATCHED_MIME_TYPES."""
```

Comment:
What is the change?

Reply:
These changes have been done via pre-commit checks while committing the code.